server/src/configs/database.js

```diff
@@ -0,0 +1,8 @@
+import mongoose from 'mongoose';
+import config from '../configs/env';
+
+const uri = config.databaseURI;
+
+mongoose.connect(uri, { useNewUrlParser: true, useUnifiedTopology: true });
```

Comment:
Have you checked this?
https://mongoosejs.com/docs/deprecations.html

Reply:
yes, after that deprecation warning i added useUnifiedtopology: true

Reply:
are createIndex errors not being thrown? Usually, I need `useCreateIndex: true` also in mongoose.connect

Reply:
not yet thrown